report/sections/data.tex

```diff
@@ -0,0 +1,71 @@
+\section{Data}\label{sec:data}
```

Comment:
In class, he laid out the report sections as:
...
Implementation
Examples
Discussion
...

We use the heading "Data" instead of "Examples", which should be fine. It's possible that online he has a slightly different outline that uses that heading instead. Either way, he wanted images/video sequences/frequency analysis etc. to be in this section which it is.